documentation/commands.md

```diff
@@ -241,3 +251,37 @@ Also:
 
 "The writer is not allowed to modify or append to any data items containing
 variable-size datatypes (including string and region references datatypes)."
+
+## Attributes
+
+Attributes are used to define metadata about the data object.
+NeXus also uses them to define NeXus classes for groups.
+
+Attributes are defined using key-value pairs, for example:
+
+```json
+"attributes": {
+  "units": "Kelvin",
+  "error": 0.02
+}
+```
+Where possible the file-writer will try to infer the type of the data, in this example: a float.
+
+For strings and arrays it is necessary to specify the type, for example:
+
+```json
```

Comment:
One final question: Do you know if it is possible to (somehow) use key-value pairs as well as arrays? If it is, how do you do that?